tests/scripts/multi-node/build-rook.sh

```diff
@@ -67,7 +67,7 @@ function run_docker_registry {
 }
 
 function docker_import {
-  img=$(docker images | grep -Eo '^build-[a-z0-9]{8}/rook-[a-z0-9]+\s')
+  img=$(docker images | grep -Eo '^build-[a-z0-9]{8}/ceph-[a-z0-9]+\s')
   # shellcheck disable=SC2086
   docker tag $img 172.17.8.1:5000/rook/rook:latest
```

Comment:
i would expect that all the `rook/rook` image references in this script should be updated to `rook/ceph` also.

there is also a `-n rook-system` reference that should be updated to `-n rook-ceph-system`.

Reply:
Duh! Can't believe i didn't see them all. thanks @jbw976. Believe i have them all sorted now.